README.md

```diff
@@ -1,4 +1,45 @@
-# Node ES Modules with TypeScript
+# Accounter
+
+Manage your taxes.
+
+## Getting started
+
+1. Switch to the correct version of Node.js:
+
+```sh
+nvm use $(cat .node-version)
+```
+
+2. Install dependencies:
+
+```sh
+yarn install
+```
+
+3. Create `.env` file:
+
+```sh
+cp .env.template .env
+```
+
+4. Run setup:
+
+```sh
+yarn local:setup
+```
+
+5. Run client and server:
+
+```sh
```

Comment:
For VSCode users, there's the `Run Accounter` button on the bottom bar

Reply:
Nice :)